playbooks/setup-ci.yml

```diff
@@ -1,11 +1,16 @@
 ---
 - hosts: ci
   remote_user: vagrant
-  sudo: yes
+  become: yes
 
   roles:
   - setlocalrepo
   - java7
   - maven
-  - { role: mysql, server-test: 1}
-  - jenkins
+  - { role: mysql, action: 'install' }
+  - { role: mysql, action: 'change-root-passwords' }
```

Comment:
Точно. Але без цього модуля видавало
помилку?
26 бер. 2016 17:50 "Igor Bronovskyi" <notifications@github.com> пише:

> In playbooks/setup-ci.yml
> <https://github.com/IF-061-DevOps/OMS-ansible/pull/56#discussion_r57515291>
> :
>
> >
> >    roles:
> >    - setlocalrepo
> >    - java7
> >    - maven
> > -  - { role: mysql, server-test: 1}
> > -  - jenkins
> > +  - { role: mysql, action: 'install' }
> > +  - { role: mysql, action: 'change-root-passwords' }
>
> install включає зміну пароля для root
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/IF-061-DevOps/OMS-ansible/pull/56/files/7a5898858d061a9d82066d5ff35e59cf4d19cc98#r57515291>
>